tests/test_container.py

```diff
@@ -133,7 +134,9 @@ def test_flow_topo_replicas(self):
             f.dry_run()
             f.index(input_fn=random_docs(1000))
 
+    @pytest.mark.skip()
```

Comment:
can you mark the reason of the skip for future reference? otherwise it's easy to become a tech debt and no one will take care of it anymore.

```suggestion
    @pytest.mark.skip()
```

Reply:
fixed.

Reply:
@hanxiao I've created two more issues on the skipped tests. I suggest to merge this PR and make the CICD working for PRs and working on the remaining issues in another PR.